packages/opentelemetry-metrics/src/export/types.ts

```diff
@@ -63,6 +63,8 @@ export interface MetricDescriptor {
   readonly type: MetricDescriptorType;
   /** The label keys associated with the metric descriptor. */
   readonly labelKeys: string[];
+  /** The label set associated with the metric descriptor. */
+  readonly labels: LabelSet;
```

Comment:
AFAIK this is not required here, we kind of de-structure  `LabelSet` into `labelKeys` (in `MetricDescriptor`) and `labelValues` (in `TimeSeries`).

Reply:
I see, thanks, will fix it.